core/templates/pages/learner-dashboard-page/progress-tab.component.html

```diff
@@ -54,8 +54,8 @@
           <span class="skill-proficiency-topic-title e2e-test-skill-proficiency-topic-title" [ngStyle]="tile[0] === 0 && {'left': '51px'}">
           {{tile[1].name}}
           </span>
-          <img [src]="getStaticImageUrl('/learner_dashboard/dropdown.svg')" alt="dropdown image" class="oppia-svg-dropdown-image" (click)="showSkills(i)" *ngIf="!displaySkills[i]">
-          <img [src]="getStaticImageUrl('/learner_dashboard/up.svg')" alt="up image" class="oppia-svg-up-image" (click)="showSkills(i)" *ngIf="displaySkills[i]">
+          <img [src]="getStaticImageUrl('/learner_dashboard/dropdown.svg')" alt="dropdown image" class="oppia-svg-dropdown-image" role="button" tabindex="0" (click)="showSkills(i)" (keydown.enter)="showSkills(i)" (click)="showSkills(i)" *ngIf="!displaySkills[i]">
```

Comment:
```suggestion
          <img [src]="getStaticImageUrl('/learner_dashboard/dropdown.svg')" alt="dropdown button" class="oppia-svg-dropdown-image" role="button" tabindex="0" (click)="showSkills(i)" (keydown.enter)="showSkills(i)" (click)="showSkills(i)" *ngIf="!displaySkills[i]">
```
It is serving as a button, so the alt text should also indicate that it is a button, rather than a image, ditto elsewhere.

Reply:
Done